lib/pfconfig/namespaces/resource/network_config.pm

```diff
@@ -0,0 +1,95 @@
+package pfconfig::namespaces::resource::network_config;
+
+=head1 NAME
+
+pfconfig::namespaces::resource::network_config
+
+=cut
+
+=head1 DESCRIPTION
+
+pfconfig::namespaces::resource::network_config
+
+=cut
+
+use strict;
+use warnings;
+use NetAddr::IP;
+use pf::util;
+
+use base 'pfconfig::namespaces::resource';
+use pfconfig::namespaces::config::Network;
+use pfconfig::namespaces::interfaces;
+
+sub init {
+    my ($self) = @_;
+
+    $self->{networks} = $self->{cache}->get_cache('config::Network');
```

Comment:
This namespace should be a dependency of config::Network

Reply:
Not sure what you mean ?
In config::Network i have that:
    $self->{child_resources} = [
        'interfaces',
        'resource::network_config'
    ];

Reply:
My bad, I must have had grease in my eyes when looking